app/scripts/lib/derived-keys.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
e.g., like you've done [here](https://github.com/mozilla/fxa-content-server/pull/2143/files#diff-93914b5a4e2d68ca8b53aa94239509c4R11).